app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        raise ValueError(
+            "Error: Invalid command format. "
+            "Use 'mv source_path destination_path'."
+        )
+
+    source_path, destination_path = parts[1], parts[2]
+
+    if not os.path.exists(source_path):
+        raise ValueError(
+            f"Error: Source file '{source_path}' not found."
+        )
+
+    if not destination_path:
+        raise ValueError(
+            "Error: Destination path is empty."
+        )
+
+    is_directory = destination_path.endswith("/")
```

Comment:
Do not use any specific directory delimiters, your code should be platform-agnostic. Use methods from the `os` module to perform similar operations.